index.html

```diff
@@ -2,9 +2,48 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>Mini Ninjas</title>
+
+    <link rel="stylesheet" href="styles/apps.css">
   </head>
   <body>
 
+
+      <div class="container1">
+        <div class="row">
+          <div class="column">
+            <figure class="cartoon">
+            <img class="cartoon_pic" src="https://octodex.github.com/images/filmtocat.png" height="400" width="400"alt="">
+            <figcaption class="cartoon_cap">
+              <p class="cartoon_number">#115</p>
+              <p class="cartoon_artist">Octocat by <strong>Matt Damon</strong> <img class="cartoon_avatar" src="http://lorempixel.com/100/100/" alt="">
+              </p>
+            </figcaption>
+          </figure>
+        </div>
+           <div class="column">
+            <figure class="cartoon">
+              <img class="cartoon_pic" src="https://octodex.github.com/images/privateinvestocat.jpg" height="400" width="400" alt="">
```

Comment:
On the `img` tags, watch out that you don't set `width` and `height` on the element. Setting these values here will make it so that the image gets squatty when the screen changes width since the `width` is overridden by CSS, but the height is set here.